roles/ansible/tower/config-ansible-tower-ocp/tasks/setup_pvc.yml

```diff
@@ -0,0 +1,52 @@
+---
+- block:
+   - name: Ensure user is authenticated with OCP
```

Comment:
This step is already performed in the `openshift_retrieve_token`, so shouldn't be needed here - probably just remove this task(?)

Reply:
I've moved the authentication part to a separate file, but in the case that only OCP token is provided we need to have a step for login. openshift_retrieve_token was only used when username and password was provided, it did not cover the token based authentication.